src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,25 @@ const createCustomerValidator = () => {
     });
 };
 
+const getCustomersValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
+            limit: Joi.string(),
```

Comment:
Poderia já receber o campo `limit` como sendo um número inteiro, e com um valor padrão caso não fosse informado.
```suggestion
            limit: Joi
                .number()
                .integer()
                .positive()
                .max(50)
                .default(50),
```